.kokoro

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright 2017-present The Material Foundation Authors. All Rights Reserved.
```

Comment:
I believe the copyright should be "the Material Components for iOS authors."

Reply:
Done.